.travis.yml

```diff
@@ -13,6 +13,7 @@ services:
 env:
   - RELEASES=2.3.0
   - RELEASES=2.3.1
+  - RELEASES=2.3.1-ubi7
```

Comment:
Note that this will only be published under the apache/couchdb space, not the top-level Docker couchdb namespace at present.